stories/cli/usability.fmf

```diff
@@ -21,6 +21,6 @@ story:
 
 /completion:
     summary: Support bash completion
-    example: tmt st<tabl> sh<tab>
+    example: tmt st<table> sh<tab>
```

Comment:
:shrug:

Reply:
Would `tab` instead of `tabl` help?

Reply:
Really don't  know. But yeah, it's supposed to be hitting a tab key? But then what's the `st`/`sh` next to each other?

Reply:
Yeah, it's supposed to be `TAB` key, the same one as used in `sh<tab>`. The sequence is supposed to complete `stories` for `st<tab>` and `show` for `sh<tab>`.

Reply:
```suggestion
    example: tmt st<tab> sh<tab>
```
If there aren't too many to change, we could use the github UI to quickly change these